ps_facetedsearch.php

```diff
@@ -93,6 +98,7 @@ public function __construct()
         $this->ps_versions_compliancy = ['min' => '1.7.1.0', 'max' => _PS_VERSION_];
 
         $this->hookDispatcher = new HookDispatcher($this);
+        $this->database = Db::getInstance();
```

Comment:
What do you think about moving this in the first call of `getDatabase()` of this class?

Reply:
Done! Thank, good idea